notebooks/4.visualization/nbconverted/4.visualization.r

```diff
@@ -166,7 +166,7 @@ fig2_B_pr_curve_plot_train_test <- ggplot(pr_f1_curve, aes(x = recall, y = preci
         legend.box.spacing = unit(0.1, "cm"),
         legend.key.size = unit(0.7, "lines"),
         legend.key.width = unit(1, "lines"),
-        axis.text.x = element_text(angle = 90, size = 20),
+        axis.text.x = element_text(angle = 90, vjust = 0.5, size = 20),
```

Comment:
I think this looks much improved! However, I recommend closing this PR and adding this commit (and reviewer comment) to #23 

(it is ok to group reviewer suggestions together, if they all have a common thread. In this case the common thread is minor improvements to figure 2)

Reply:
Ah! That's a good point! I'll add it to that PR. Thanks Greg! Closing c: !